views/account/profile/main.js

```diff
@@ -0,0 +1,56 @@
+// JQuery Datatables Initialization
+$(document).ready(function () {
+  var table = $("#casual-table").DataTable();
+  new $.fn.DataTable.Responsive(table);
+});
+$(document).ready(function () {
+  var table = $("#professional-table").DataTable();
+  new $.fn.DataTable.Responsive(table);
+});
+
```

Comment:
I'm also generally concerned about having a paginator and a search bar implemented in this way because it will be on a mobile version. 

Actually, have you seen the mockups Payton drew here? https://docs.google.com/document/d/1H53wnfOcuWR4XAgfxG4HXK4YMwiOpZ_o_YYnFOY90-M/edit

Reply:
> Actually, have you seen the mockups Payton drew here? https://docs.google.com/document/d/1H53wnfOcuWR4XAgfxG4HXK4YMwiOpZ_o_YYnFOY90-M/edit

Yeah, many of the comments that I made assumed that we were using what Leah implemented, but if we go the way of the quick design I made then things like an edit or create modal wouldn't be needed because they do all of that in place.

Reply:
Keeping in mind we have pretty much a month left to implement this, I wouldn't be opposed to moving forward with the table format but I'd like to consider the different options. Though I'm not sure what would be the best approach to implement this here.

Reply:
I can try to make it look more like the mockup; I didn't know that this drawing existed lol. I can try to do something similar looking to that later this week!